tests/data/issues/1540

```diff
@@ -1,5 +1,15 @@
 {
   "number": 1540,
   "title": "A mock open issue for testing",
-  "state": "open"
+  "state": "open",
+  "labels": [
+      {
+        "name": "refactor",
+        "color": "47f441"
+      },
+      {
+        "name": "bug",
+        "color": "f4425c"
+      }
```

Comment:
For completeness, we should add a `"color": ""` test case, and a `"color": "this is not a color"` test case.

Reply:
Couldn't understand this :disappointed:

Reply:
A good option might be to have a fallback color for when things go wrong. E.g., white text, black background.

Reply:
Ah fine :smiley:

Reply:
I have pushed a part of the changes now. I am adding this feature now